tools/aspnet_compiler/StaticCompilationService.cs

```diff
@@ -73,21 +73,21 @@ private sealed class PersistedCompilation(StaticCompilationOptions options) : IC
         public bool HandleExceptions => false;
 
         Stream ICompilationStrategy.CreatePdbStream(string route, string typeName)
```

Comment:
hmmm if this works, that seems like a coincidence. Maybe add a new parameter "assemblyName" to the ICompilationStrategy to make sure we're keeping everything in sync.

Reply:
I initially used this method as it required the least amount of changes.  I can redo it to modify `CreatePdbStream` and `CreatePeStream` to pass in an assembly name instead.  At the point it is used in `SystemWebCompilation.CreateCompiledPage`, `compilation.Assemblyname` is available, so it should be simple.

Reply:
sounds good - I imagine we can simplify it later, but for now more explicit the better :)